### PR TITLE
RMET-1182 ::: Fix - Ciphered Database Errors

### PR DESCRIPTION
**Pull request by CarlsCorrea**
## Description

Fix for the error messages:
Keystore operation failed
User not authenticated 
Key not yet valid

Work done in colaboration with @OS-ricardomoreirasilva 

## Context

references: https://outsystemsrd.atlassian.net/browse/RMET-1182

## Type of changes

- [x] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [x] Android

## Tests

Manual Testes executed and validated with the customer in the support case: https://outsystemsrd.atlassian.net/browse/RMET-1182

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [ ] Code follows code style of this project
- [ ] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

**Review by OS-ricardomoreirasilva — APPROVED** [review]

File: src/android/RSA.java [hunk 1]
```diff
@@ -80,8 +73,6 @@ public static void createKeyPair(Context ctx, String alias) throws Exception {
 						.setAlias(alias)
 						.setSubject(new X500Principal(principalString))
 						.setSerialNumber(BigInteger.ONE)
-						.setStartDate(notBefore.getTime())
```
Comment (alexgerardojacinto) [inline src/android/RSA.java:83]: Just trying to understand why we don't use startDate and endDate anymore. Apologies in advance as I may be lacking context

Comment (CarlsCorrea) [reply src/android/RSA.java:83]: Because of the error "Key not yet valid" we are removing the validity of the keys. Its not needed because the default is always valid.

Comment (alexgerardojacinto) [reply src/android/RSA.java:83]: ok ok got it, thanks!

**Review by alexgerardojacinto — APPROVED** [review]

File: src/android/RSA.java [hunk 2]
```diff
@@ -64,11 +61,7 @@ public static void createKeyPair(Context ctx, String alias) throws Exception {
 						.setDigests(KeyProperties.DIGEST_SHA256, KeyProperties.DIGEST_SHA512)
 						.setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_RSA_PKCS1)
 						.setRandomizedEncryptionRequired(true)
-						.setKeyValidityStart(notBefore.getTime())
-						.setKeyValidityEnd(notAfter.getTime());
-				if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.N){
```
Comment (usernuno) [inline src/android/RSA.java:69]: Why was this deleted? The method only exists from that API forward IIRC.

Comment (CarlsCorrea) [reply src/android/RSA.java:69]: Already talked in slack but to register, we removed this if because we have another if before: if(Build.VERSION.SDK_INT > Build.VERSION_CODES.M)
